app/src/main/res/values-night/styles.xml

```diff
@@ -0,0 +1,6 @@
+<resources>
+    <style name="Widget.Theme.Wizards.MyView" parent="">
```

Comment:
when creating a new style, it should use the app theme as parent. Inside `Manifest` you can look up what we currently use as app theme.
But I think this style is not necessary for feature main screen/discover screen. Please remove.